WORKSPACE

```diff
@@ -136,6 +136,13 @@ go_repository(
     version = "v0.0.0-20221024183307-1bc688fe9f3e",
 )
 
+go_repository(
+    name = "org_golang_google_grpc_cmd_protoc_gen_go_grpc",
```

Comment:
This is referenced in BUILD files, so we would either need to set up the release updater for it or explicitly document that users have to provide it themselves.

Cc @linzhp as I'm not as firm on the state of our proto deps.

Reply:
@linzhp friendly ping for your input

Reply:
Yes, this is used by `go_grpc` compiler and will be depended by users of rules_go. We need to move this to `go/private/repositories.bzl` so it can be updated by releaser and loaded by users automatically, together with `org_golang_google_protobuf` etc.

Reply:
@linzhp I just removed it from there after the https://github.com/bazelbuild/rules_go/pull/3761#discussion_r1410484447 from @fmeum ...

Reply:
Done.